src/ripple/app/tx/impl/InvariantCheck.h

```diff
@@ -342,6 +353,19 @@ class ValidNFTokenPage
         beast::Journal const&);
 };
 
+/**
+ * @brief Invariant: Validates counts of NFTokens after all transaction types.
+ *
+ * The following checks are made:
+ *  - The number of minted or burned tokens may only be changed by
+ *    NFTokenMint or NFTokenBurn transactions.
+ *  - A successful NFTokenMint must increase the number of NFTokens.
+ *  - A failed NFTokenMint must not change the number of minted NFTokens.
+ *  - An NFTokenMint transaction may not change the number of burned NFTokens,
```

Comment:
"An NFTokenMint transaction cannot change the number of burned NFTokens..." or "An NFTokenMint transaction must not change the number of burned NFTokens...."

One reason for this is that you might actually mean that the transaction _might_ or _might not_ change the number of burned NFTokens.

Also, the other lines end with a period, rather than a comma.